### PR TITLE
result for each trigger

### PR DESCRIPTION
**Pull request by pnuckowski**
- [x] add execution time
- [x] get expected number of hosts from trigger

### CODE REVIEW

**Comment by ch40s3k** [comment]
:+1:

**Comment by pawellesniewski** [comment]
:ok_woman:

File: alamo_worker/plugins/__init__.py [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
+import pytz
```
Comment (deejay1) [inline alamo_worker/plugins/__init__.py]: make sure it's in requirements

**Comment by deejay1** [comment]
Also can you add the test for https://coveralls.io/builds/4587194/source?filename=alamo_worker%2Fplugins%2Fhttp_check.py#L45 ?

**Comment by deejay1** [comment]
:+1:

**Pull request merged**